src/PrestaShopBundle/Resources/config/services/adapter/image.yml

```diff
@@ -17,6 +17,7 @@ services:
       class: 'PrestaShop\PrestaShop\Adapter\Image\Uploader\EmployeeImageUploader'
       arguments:
         - !php/const _PS_EMPLOYEE_IMG_DIR_
+        - !php/const _PS_TMP_IMG_DIR_
```

Comment:
No modif on EmployeeImageUploader , the argument was missing?

Reply:
Yes, I forgot it in the previous PR.